testadd.py

```diff
@@ -0,0 +1,5 @@
+from fastapi import FastAPI
```

Comment:
if you use `testadd` as a name for this file then it would be confusing. Instead name appropriately, eg. MathUtils.
Crate a package and move this file under that package. Follow some best practices to get use to it.